### DISCUSSION TITLE
IDE (WebStorm) warning when trying to apply placeholder text color

### DISCUSSION THREAD

**Discussion by CorvS** [nativewind/nativewind#1359] [General]
Hello everyone, 👋 

I was wondering if anyone else has encountered the same problem when trying to apply a placeholder color to a text input.

When I use
```
<TextInput className="text-black placeholder:text-gray-400"  />
```
for example. 
WebStorm shows me the warning
**'text-black' applies the same CSS properties as 'placeholder:text-gray-400'.**

It still works as expected, but the warning still triggers my OCD and I'd like to get rid of it. 😄
Of course, I could just use the `placeholderTextColor` attribute directly, but that IMO interrupts the flow when working with Tailwind.

I use Tailwind for pretty much all my web projects, but have never encountered this problem, so I thought it might be related to NativeWind and I might have missed something.

Cheers!

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
